npm install failing on windows

```
C:\Users\Owner\Documents\bb-toto-bounce-back>vagrant up
Bringing machine 'default' up with 'virtualbox' provider...
==> default: Importing base box 'ubuntu/trusty64'...
==> default: Matching MAC address for NAT networking...
==> default: Checking if box 'ubuntu/trusty64' is up to date...
==> default: Setting the name of the VM: bb-toto-bounce-back_default_1507318200206_38874
==> default: Clearing any previously set forwarded ports...
==> default: Clearing any previously set network interfaces...
==> default: Preparing network interfaces based on configuration...
    default: Adapter 1: nat
==> default: Forwarding ports...
    default: 22 (guest) => 2222 (host) (adapter 1)
==> default: Booting VM...
==> default: Waiting for machine to boot. This may take a few minutes...
    default: SSH address: 127.0.0.1:2222
    default: SSH username: vagrant
    default: SSH auth method: private key
    default: Warning: Connection aborted. Retrying...
    default: Warning: Remote connection disconnect. Retrying...
    default: Warning: Connection aborted. Retrying...
    default:
    default: Vagrant insecure key detected. Vagrant will automatically replace
    default: this with a newly generated keypair for better security.
    default:
    default: Inserting generated public key within guest...
    default: Removing insecure key from the guest if it's present...
    default: Key inserted! Disconnecting and reconnecting using new SSH key...
==> default: Machine booted and ready!
==> default: Checking for guest additions in VM...
    default: The guest additions on this VM do not match the installed version of
    default: VirtualBox! In most cases this is fine, but in rare cases it can
    default: prevent things such as shared folders from working properly. If you see
    default: shared folder errors, please make sure the guest additions within the
    default: virtual machine match the version of VirtualBox you have installed on
    default: your host and reload your VM.
    default:
    default: Guest Additions Version: 4.3.36
    default: VirtualBox Version: 5.1
==> default: Mounting shared folders...
    default: /vagrant => C:/Users/Owner/Documents/bb-toto-bounce-back
==> default: Running provisioner: ansible_local...
    default: Installing Ansible...
Vagrant has automatically selected the compatibility mode '2.0'
according to the Ansible version installed (2.4.0.0).

Alternatively, the compatibility mode can be specified in your Vagrantfile:
https://www.vagrantup.com/docs/provisioning/ansible_common.html#compatibility_mode
    default: Running ansible-playbook...
[DEPRECATION WARNING]: The use of 'include' for tasks has been deprecated. Use
'import_tasks' for static inclusions or 'include_tasks' for dynamic inclusions.
 This feature will be removed in a future release. Deprecation warnings can be
disabled by setting deprecation_warnings=False in ansible.cfg.
[DEPRECATION WARNING]: include is kept for backwards compatibility but usage is
 discouraged. The module documentation details page may explain more about this
 rationale.. This feature will be removed in a future release. Deprecation
warnings can be disabled by setting deprecation_warnings=False in ansible.cfg.
[DEPRECATION WARNING]: The use of 'static' has been deprecated. Use
'import_tasks' for static inclusion, or 'include_tasks' for dynamic inclusion.
This feature will be removed in a future release. Deprecation warnings can be
disabled by setting deprecation_warnings=False in ansible.cfg.

PLAY [common configuration for all systems] ************************************

TASK [Gathering Facts] *********************************************************
ok: [default]

TASK [geerlingguy.git : Ensure git is installed (RedHat).] *********************
skipping: [default] => (item=git)
skipping: [default] => (item=git-svn)

TASK [geerlingguy.git : Update apt cache (Debian).] ****************************
ok: [default]

TASK [geerlingguy.git : Ensure git is installed (Debian).] *********************
changed: [default] => (item=[u'git', u'git-svn'])

TASK [geerlingguy.git : Include OS-specific variables (RedHat).] ***************
skipping: [default]

TASK [geerlingguy.git : Include OS-specific variables (Fedora).] ***************
skipping: [default]

TASK [geerlingguy.git : Include OS-specific variables (Debian).] ***************
skipping: [default]

TASK [geerlingguy.git : Define git_install_from_source_dependencies.] **********
skipping: [default]

TASK [geerlingguy.git : Ensure git's dependencies are installed (RedHat).] *****
skipping: [default]

TASK [geerlingguy.git : Ensure git's dependencies are installed (Debian).] *****
skipping: [default]

TASK [geerlingguy.git : Get installed version] *********************************
skipping: [default]

TASK [geerlingguy.git : Force git install if the version numbers do not match] ***
skipping: [default]

TASK [geerlingguy.git : Download git.] *****************************************
skipping: [default]

TASK [geerlingguy.git : Expand git archive.] ***********************************
skipping: [default]

TASK [geerlingguy.git : Build git.] ********************************************
skipping: [default] => (item=all)
skipping: [default] => (item=install)

TASK [geerlingguy.nodejs : Set up the Nodesource RPM directory for Node.js > 0.10.] ***
skipping: [default]

TASK [geerlingguy.nodejs : Set up the Nodesource RPM variable for Node.js == 0.10.] ***
skipping: [default]

TASK [geerlingguy.nodejs : Import Nodesource RPM key (CentOS < 7).] ************
skipping: [default]

TASK [geerlingguy.nodejs : Import Nodesource RPM key (CentOS 7+)..] ************
skipping: [default]

TASK [geerlingguy.nodejs : Add Nodesource repositories for Node.js (CentOS < 7).] ***
skipping: [default]

TASK [geerlingguy.nodejs : Add Nodesource repositories for Node.js (CentOS 7+).] ***
skipping: [default]

TASK [geerlingguy.nodejs : Ensure Node.js and npm are installed.] **************
skipping: [default]

TASK [geerlingguy.nodejs : Ensure apt-transport-https is installed.] ***********
ok: [default]

TASK [geerlingguy.nodejs : Add Nodesource apt key.] ****************************
changed: [default]

TASK [geerlingguy.nodejs : Add NodeSource repositories for Node.js.] ***********
changed: [default] => (item=deb https://deb.nodesource.com/node_6.x trusty main)
changed: [default] => (item=deb-src https://deb.nodesource.com/node_6.x trusty main)

TASK [geerlingguy.nodejs : Update apt cache if repo was added.] ****************
changed: [default]

TASK [geerlingguy.nodejs : Ensure Node.js and npm are installed.] **************
changed: [default]

TASK [geerlingguy.nodejs : Define nodejs_install_npm_user] *********************
ok: [default]

TASK [geerlingguy.nodejs : Create npm global directory] ************************
changed: [default]

TASK [geerlingguy.nodejs : Add npm_config_prefix bin directory to global $PATH.] ***
changed: [default]

TASK [geerlingguy.nodejs : Ensure npm global packages are installed.] **********

TASK [geerlingguy.nodejs : Install packages defined in a given package.json.] ***
skipping: [default]

TASK [geerlingguy.mysql : Include OS-specific variables.] **********************
ok: [default] => (item=/vagrant/roles/geerlingguy.mysql/vars/Debian.yml)

TASK [geerlingguy.mysql : Include OS-specific variables (RedHat).] *************
skipping: [default]

TASK [geerlingguy.mysql : Define mysql_packages.] ******************************
ok: [default]

TASK [geerlingguy.mysql : Define mysql_daemon.] ********************************
ok: [default]

TASK [geerlingguy.mysql : Define mysql_slow_query_log_file.] *******************
ok: [default]

TASK [geerlingguy.mysql : Define mysql_log_error.] *****************************
ok: [default]

TASK [geerlingguy.mysql : Define mysql_syslog_tag.] ****************************
ok: [default]

TASK [geerlingguy.mysql : Define mysql_pid_file.] ******************************
ok: [default]

TASK [geerlingguy.mysql : Define mysql_config_file.] ***************************
ok: [default]

TASK [geerlingguy.mysql : Define mysql_config_include_dir.] ********************
ok: [default]

TASK [geerlingguy.mysql : Define mysql_socket.] ********************************
ok: [default]

TASK [geerlingguy.mysql : Define mysql_supports_innodb_large_prefix.] **********
ok: [default]

TASK [geerlingguy.mysql : include] *********************************************
skipping: [default]

TASK [geerlingguy.mysql : include] *********************************************
included: /vagrant/roles/geerlingguy.mysql/tasks/setup-Debian.yml for default

TASK [geerlingguy.mysql : Check if MySQL is already installed.] ****************
ok: [default]

TASK [geerlingguy.mysql : Update apt cache if MySQL is not yet installed.] *****
changed: [default]

TASK [geerlingguy.mysql : Ensure MySQL Python libraries are installed.] ********
changed: [default]

TASK [geerlingguy.mysql : Ensure MySQL packages are installed.] ****************
changed: [default] => (item=[u'mysql-common', u'mysql-server'])

TASK [geerlingguy.mysql : Ensure MySQL is stopped after initial install.] ******
changed: [default]

TASK [geerlingguy.mysql : Delete innodb log files created by apt package after initial install.] ***
changed: [default] => (item=ib_logfile0)
changed: [default] => (item=ib_logfile1)

TASK [geerlingguy.mysql : include] *********************************************
skipping: [default]

TASK [geerlingguy.mysql : Check if MySQL packages were installed.] *************
ok: [default]

TASK [geerlingguy.mysql : Copy my.cnf global MySQL configuration.] *************
changed: [default]

TASK [geerlingguy.mysql : Verify mysql include directory exists.] **************
skipping: [default]

TASK [geerlingguy.mysql : Copy my.cnf override files into include directory.] ***

TASK [geerlingguy.mysql : Create slow query log file (if configured).] *********
skipping: [default]

TASK [geerlingguy.mysql : Create datadir if it does not exist] *****************
changed: [default]

TASK [geerlingguy.mysql : Set ownership on slow query log file (if configured).] ***
skipping: [default]

TASK [geerlingguy.mysql : Create error log file (if configured).] **************
changed: [default]

TASK [geerlingguy.mysql : Set ownership on error log file (if configured).] ****
changed: [default]

TASK [geerlingguy.mysql : Ensure MySQL is started and enabled on boot.] ********
changed: [default]

TASK [geerlingguy.mysql : Get MySQL version.] **********************************
ok: [default]

TASK [geerlingguy.mysql : Ensure default user is present.] *********************
skipping: [default]

TASK [geerlingguy.mysql : Copy user-my.cnf file with password credentials.] ****
skipping: [default]

TASK [geerlingguy.mysql : Disallow root login remotely] ************************
ok: [default] => (item=DELETE FROM mysql.user WHERE User='root' AND Host NOT IN ('localhost', '127.0.0.1', '::1'))

TASK [geerlingguy.mysql : Get list of hosts for the root user.] ****************
ok: [default]

TASK [geerlingguy.mysql : Update MySQL root password for localhost root account (5.7.x).] ***
skipping: [default] => (item=127.0.0.1)
skipping: [default] => (item=::1)
skipping: [default] => (item=localhost)

TASK [geerlingguy.mysql : Update MySQL root password for localhost root account (< 5.7.x).] ***
changed: [default] => (item=127.0.0.1)
changed: [default] => (item=::1)
changed: [default] => (item=localhost)

TASK [geerlingguy.mysql : Copy .my.cnf file with root password credentials.] ***
changed: [default]

TASK [geerlingguy.mysql : Get list of hosts for the anonymous user.] ***********
ok: [default]

TASK [geerlingguy.mysql : Remove anonymous MySQL users.] ***********************

TASK [geerlingguy.mysql : Remove MySQL test database.] *************************
ok: [default]

TASK [geerlingguy.mysql : Ensure MySQL databases are present.] *****************

TASK [geerlingguy.mysql : Ensure MySQL users are present.] *********************

TASK [geerlingguy.mysql : Ensure replication user exists on master.] ***********
skipping: [default]

TASK [geerlingguy.mysql : Check slave replication status.] *********************
skipping: [default]

TASK [geerlingguy.mysql : Check master replication status.] ********************
skipping: [default]

TASK [geerlingguy.mysql : Configure replication on the slave.] *****************
skipping: [default]

TASK [geerlingguy.mysql : Start replication.] **********************************
skipping: [default]

TASK [install node packages for bot] *******************************************
fatal: [default]: FAILED! => {"changed": false, "cmd": "/usr/bin/npm install", "failed": true, "msg": "npm WARN enoent ENOENT:********@1.0.0 No repository field.\nnpm ERR! Linux 3.13.0-132-generic\nnpm ERR! argv \"/usr/bin/node\" \"/usr/bin/npm\" \"install\"\nnpm ERR! node v6.11.4\nnpm ERR! npm  v3.10.10\nnpm ERR! path ../mkdirp/bin/cmd.js\nnpm ERR! code EPROTO\nnpm ERR! errno -71\nnpm ERR! syscall symlink\n\nnpm ERR! EPROTO: protocol error, symlink '../mkdirp/bin/cmd.js' -> '/vagrant/node_modules/sqlite3/node_modules/.bin/mkdirp'\nnpm ERR! \nnpm ERR! If you need help, you may report this error at:\nnpm ERR!     <https://github.com/npm/npm/issues>\n\nnpm ERR! Please include the following file with any support request:\nnpm ERR!     /vagrant/npm-debug.log", "rc": 185, "stderr": "npm WARN enoent ENOENT: no such file or directory, open '/vagrant/node_modules/json-rpc2/package.json'\nnpm WARN conditional-token-sale@1.0.0 No repository field.\nnpm ERR! Linux 3.13.0-132-generic\nnpm ERR! argv \"/usr/bin/node\" \"/usr/bin/npm\" \"install\"\nnpm ERR! node v6.11.4\nnpm ERR! npm  v3.10.10\nnpm ERR! path ../mkdirp/bin/cmd.js\nnpm ERR! code EPROTO\nnpm ERR! errno -71\nnpm ERR! syscall symlink\n\nnpm ERR! EPROTO: protocol error, symlink '../mkdirp/bin/cmd.js' -> '/vagrant/node_modules/sqlite3/node_modules/.bin/mkdirp'\nnpm ERR! \nnpm ERR! If you need help, you may report this error at:\nnpm ERR!     <https://github.com/npm/npm/issues>\n\nnpm ERR! Please include the following file with any support request:\nnpm ERR!     /vagrant/npm-debug.log\n", "stderr_lines": ["npm WARN enoent ENOENT: no such file or directory, open '/vagrant/node_modules/json-rpc2/package.json'", "npm WARN conditional-token-sale@1.0.0 No repository field.", "npm ERR! Linux 3.13.0-132-generic", "npm ERR! argv \"/usr/bin/node\" \"/usr/bin/npm\" \"install\"", "npm ERR! node v6.11.4", "npm ERR! npm  v3.10.10", "npm ERR! path ../mkdirp/bin/cmd.js", "npm ERR! code EPROTO", "npm ERR! errno -71", "npm ERR! syscall symlink", "", "npm ERR! EPROTO: protocol error, symlink '../mkdirp/bin/cmd.js' -> '/vagrant/node_modules/sqlite3/node_modules/.bin/mkdirp'", "npm ERR! ", "npm ERR! If you need help, you may report this error at:", "npm ERR!     <https://github.com/npm/npm/issues>", "", "npm ERR! Please include the following file with any support request:", "npm ERR!     /vagrant/npm-debug.log"], "stdout": "conditional-token-sale@1.0.0 /vagrant\nâ"oâ"?â"? async@2.5.0 \nâ"oâ"?â"¬ emailjs@1.0.12 \nâ"' â""â"?â"? starttls@1.0.1 \nâ"oâ"?â"? moment@2.18.1 \nâ""â"?â"¬ request@2.83.0 \n  â"oâ"?â"¬ http-signature@1.2.0 \n  â"' â"oâ"?â"¬ jsprim@1.4.1 \n  â"' â"' â""â"?â"? verror@1.10.0 \n  â"' â""â"?â"¬ sshpk@1.13.1 \n  â"'   â"oâ"?â"? bcrypt-pbkdf@1.0.1 \n  â"'   â""â"?â"? tweetnacl@0.14.5 \n  â"oâ"?â"? stringstream@0.0.5 \n  â"oâ"?â"? tough-cookie@2.3.3 \n  â"oâ"?â"? tunnel-agent@0.6.0 \n  â""â"?â"? uuid@3.1.0 \n\n", "stdout_lines": ["conditional-token-sale@1.0.0 /vagrant", "â"oâ"?â"? async@2.5.0 ", "â"oâ"?â"¬ emailjs@1.0.12 ", "â"' â""â"?â"? starttls@1.0.1 ", "â"oâ"?â"? moment@2.18.1 ", "â""â"?â"¬ request@2.83.0 ", "  â"oâ"?â"¬ http-signature@1.2.0 ", "  â"' â"oâ"?â"¬ jsprim@1.4.1 ", "  â"' â"' â""â"?â"? verror@1.10.0 ", "  â"' â""â"?â"¬ sshpk@1.13.1 ", "  â"'   â"oâ"?â"? bcrypt-pbkdf@1.0.1 ", "  â"'   â""â"?â"? tweetnacl@0.14.5 ", "  â"oâ"?â"? stringstream@0.0.5 ", "  â"oâ"?â"? tough-cookie@2.3.3 ", "  â"oâ"?â"? tunnel-agent@0.6.0 ", "  â""â"?â"? uuid@3.1.0 ", ""]}

RUNNING HANDLER [geerlingguy.mysql : restart mysql] ****************************
        to retry, use: --limit @/vagrant/playbook.retry

PLAY RECAP *********************************************************************
default                    : ok=42   changed=19   unreachable=0    failed=1

Ansible failed to complete successfully. Any error output should be
visible above. Please fix these errors and try again.
```

--- a/.gitignore
+++ b/.gitignore
@@ -1,3 +1,4 @@
 .vagrant
 playbook.retry
 node_modules
+npm-debug.log

--- a/playbook.yml
+++ b/playbook.yml
@@ -11,5 +11,6 @@
 
   tasks:
     - name: install node packages for bot
-      npm:
-        path: /vagrant
+      command: npm install --no-bin-links
+      args:
+        chdir: /vagrant

